src/Shared/Process/ProcessEx.cs

```diff
@@ -0,0 +1,210 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Why is this file shared? Seems to be used in only one project.

Reply:
I took it from /src/ProjectTemplates/Helpers. Will update it to use the shared file before merging.

Reply:
Deleted file from templates project so they use the same shared file.